cron/createFDQNBlockList.py

```diff
@@ -121,7 +121,7 @@ def initConfig():
 
     # mongo instance
     options.mongohost = getConfig('mongohost', 'localhost', options.configfile)
-    options.mongoport = getConfig('mongoport', 3001, options.configfile)
+    options.mongoport = getConfig('mongoport', 3002, options.configfile)
```

Comment:
I don't know that these changes are necessary for this PR, so in the spirit of trying to minimize changes, I think we should keep the original default value.

Reply:
Resolved in a [recent commit](https://github.com/mozilla/MozDef/pull/1576/commits/775a593f8c59f00026d4ff96e955d56b6a82a8bf).